addons/web_editor/static/src/js/editor/odoo-editor/src/utils/utils.js

```diff
@@ -1868,14 +1868,15 @@ export function getFontSizeDisplayValue(sel, getCSSVariableValue, convertNumeric
     let remValue;
     if (closestFontSizedEl) {
         const useFontSizeInput = closestFontSizedEl.style.fontSize;
+        const match = useFontSizeInput.match(/^([\d.]+)([a-zA-Z]+)/);
```

Comment:
This should be one line lower, to call `match` on `useFontSizeInput` once we know `useFontSizeInput` is defined.